### ISSUE TITLE
implode() use causes a false positive

### ISSUE THREAD

**Issue by johnbillion** [WordPress-Coding-Standards/WordPress-Coding-Standards#442]
The following code triggers an `Expected a sanitizing function...` warning, despite the output of `implode()` being safe:

```php
$items = array( 'foo', 'bar' );
echo implode( '<br>', esc_html( $items ) );
```

**Comment by johnbillion** [comment]
Just noticed that my simplified example code is wrong because you can't call `esc_html()` on an array, but you get the idea.
